src/Interface/InterfaceForm.js

```diff
@@ -71,10 +72,24 @@ class InterfaceForm extends Component {
             <Field label={<FormattedMessage id="ui-organizations.interface.available" />} name={`${elem}.available`} id={`${elem}.available`} component={Checkbox} />
           </Col>
           <Col xs={12} md={6}>
-            <Field label={<FormattedMessage id="ui-organizations.interface.deliveryMethod" />} name={`${elem}.deliveryMethod`} id={`${elem}.deliveryMethod`} component={Select} fullWidth dataOptions={deliveryMethodDD} />
+            <Field
+              label={<FormattedMessage id="ui-organizations.interface.deliveryMethod" />}
+              name={`${elem}.deliveryMethod`}
+              id={`${elem}.deliveryMethod`}
```

Comment:
I'm not sure if we need these `ids`. Since you're touching the code right now, maybe it makes sense to remove them?